galaxy_api/api/ui/serializers/collection.py

```diff
@@ -50,26 +61,26 @@ class CollectionVersionBaseSerializer(serializers.Serializer):
     namespace = serializers.CharField()
     name = serializers.CharField()
     version = serializers.CharField()
-    created_at = serializers.DateTimeField(source='_created')
 
+    created_at = serializers.DateTimeField(source='_created')
 
 class CollectionLatestVersionSerializer(CollectionVersionBaseSerializer):
     metadata = CollectionMetadataBaseSerializer(source='*')
 
+class CollectionLatestVersionDetailSerializer(CollectionLatestVersionSerializer):
+    docs_blob = serializers.JSONField()
+    contents = serializers.ListField(ContentSerializer())
```

Comment:
I need `contents` to get returned in the list view.

Reply:
And the list view `['latest_version']['contents']` item models are the same as for the collection detail view?  

But the 'contents' in detail view ( ['latest_version']['docs_blob']['contents'] ) is different?

ie, list view `['latest_version']['contents']`:

``` json
            "latest_version": {
                "contents": [
                    {
                        "content_type": "module",
                        "description": null,
                        "name": "qradar_deploy"
                    },
                   " #  ITEMS TRIMMED FOR BREVITY ",
                    {
                        "content_type": "module_utils",
                        "description": null,
                        "name": "qradar"
                    },
                    {
                        "content_type": "httpapi",
                        "description": "HttpApi Plugin for IBM QRadar",
                        "name": "qradar"
                    }
                ],
```

detail view ['latest_content']['contents']:

``` json
    "latest_version": {
        "contents": [
            {
                "content_type": "module",
                "description": null,
                "name": "qradar_deploy"
            },
            " # ITEMS TRIMMED FOR BREVITY",
            {
                "content_type": "module_utils",
                "description": null,
                "name": "qradar"
            },
            {
                "content_type": "httpapi",
                "description": "HttpApi Plugin for IBM QRadar",
                "name": "qradar"
            }
        ],
```

But different from the 'contents' field in detail view ['latest_version']['docs_blob']['contents']

``` json
    "latest_version": {
        "docs_blob": {
            "contents": [
                {
                    "content_name": "qradar_deploy",
                    "content_type": "module",
                    "doc_strings": null,
                    "readme_file": null,
                    "readme_html": null
                },
                " # ITEMS SKIPPED FOR BREVITY",
                {
                    "content_name": "qradar",
                    "content_type": "module_utils",
                    "doc_strings": null,
                    "readme_file": null,
                    "readme_html": null
                },
                {
                    "content_name": "qradar",
                    "content_type": "httpapi",
                    "doc_strings": {
                        "doc": {
                            "author": "Ansible Security Automation Team",
                            "description": [
                                "This HttpApi plugin provides methods to connect to IBM QRadar over a HTTP(S)-based api."
                            ],
                            "filename": "/tmp/tmp5iifq9uu/plugins/httpapi/qradar.py",
                            "httpapi": "qradar",
                            "short_description": "HttpApi Plugin for IBM QRadar",
                            "version_added": "2.9"
                        },
                        "examples": null,
                        "metadata": {
                            "status": [
                                "preview"
                            ],
                            "supported_by": "community"
                        },
                        "return": null
                    },
                    "readme_file": null,
                    "readme_html": null
                }
```

Reply:
I think e371142d13d8449fd4dae7b90b75855876c6ede1 should add 'contents' to both list and detail views.